### ISSUE TITLE
Research Visualization Tools

### ISSUE THREAD

**Issue by de-la-viz** [codeforboston/safe-water#88]
What are the options, their advantages and disadvantages? So that when the UX research is completed we can select the potential solutions faster. 
e.g. D3, Tableau, Datawrapper, Python Dash,  etc.

**Comment by de-la-viz** [comment]
First exploration by Kevin there:
https://docs.google.com/spreadsheets/d/1rjUZOSPbeHR_pk0cvknAEXi3r6OYKeU5fRjzAMHxmiA

**Comment by de-la-viz** [comment]
findings: https://docs.google.com/presentation/d/1EeefX6x2VNSdfLWhFgB7GkKxTRFPZQ_VBlluOlIrtv4/

**Comment by de-la-viz** [comment]
We went with ArcGIS because:
1. CRWA already pays for it
2. are familiar with it; for instance, use it already to create the static map on https://www.crwa.org/water-quality-data.html to present monthly E.Coli data
3. we decided, together with CRWA, that what we want, at first, is an exploratory tool. Allow users to explore the data. The data being localized, with samples taken all along the river, we want to be able to create a connection with the users, and allow them to check what's going on in their local town while also offering the possibility to have an overview of the full watershed and how it looks like (e.g. affluents to the Charles too) ==> a map should take a central place on the dashboard.
4. the focus on CRWA being able to run, maintain, update, and further develop the visualization is a major priority for this project.

In retrospective: 
As we discover, learn, and use ArcGIS online and its plotting capabilities, we realize that it is very limited...
